soc/riscv/litex-vexriscv/Kconfig.soc

```diff
@@ -6,6 +6,9 @@ config SOC_RISCV32_LITEX_VEXRISCV
 	select RISCV
 	select ATOMIC_OPERATIONS_C
 	select INCLUDE_RESET_VECTOR
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
```

Comment:
I wonder if using `RISCV_ISA_EXT_A` wouldn't make the `ATOMIC_OPERATIONS_C` kinda redundant? So those would be better served by `ATOMIC_OPERATIONS_BUILTIN`?

Reply:
I don't think so. `RISCV_ISA_EXT_A` is specifying an extension supported by the core, `ATOMIC_OPERATIONS_BUILTIN` is referring to the usage of the compiler builtin functions.